tasks.php

```diff
@@ -820,9 +821,8 @@ function handle_action_on_a_specified_task()
         }
         mysqli_free_result($meTooCheck);
 
-        // No need to display a different error message if the user was refreshing
-        ShowNotification("Thank you for your report!  It has been recorded below.", false, "info");
-        TaskDetails($task_id);
+        // Redirect back to show task page to clear POST data
+        metarefresh(0,"$tasks_url?action=show&task_id=$task_id");
```

Comment:
Please add a space after the comma. Go ahead and squash the fixup and we'll get his merged.

Reply:
Done :)